src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4088,6 +4088,7 @@ internal static string GetClearHostFunctionText()
                 // Porting note: non-Windows platforms use `clear`
                 return @"
 & (Get-Command -CommandType Application clear | Select-Object -First 1).Definition
```

Comment:
It probably won't matter in practice, but `tput clear` is the POSIX-compliant way to clear the screen. Switching to that would allow us to get rid of the awkward `Get-Command` solution, given that no alias then needs to be bypassed.